### PR TITLE
feat: created the step 1 component for the tracked repository wizard for workspaces

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Introduces the first step component of the Tracked Repositories wizard for workspace creation/edition.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
 
- Relates to #2482
- [Designs in Figma](https://www.figma.com/file/sqRXS7hw6E93QgRMCl3WYz/Workspaces?type=design&node-id=634-11378&mode=dev)

## Mobile & Desktop Screenshots/Recordings

**It's not reflected in the video but the icon has change to the GH icon for importing repos from an org.**

https://github.com/open-sauced/app/assets/833231/9f1f5114-0fdb-4b6e-b088-1131a2f6580b

CleanShot 2024-01-17 at 09 37 34

## Steps to QA

N/A but you can [view the component in Storybook](https://deploy-preview-2481--design-insights.netlify.app/?path=/docs/components-workspaces-trackedrepowizardstep1--docs).

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/Workspaces/TrackedRepoWizardStep1.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              <span data-button-title className="font-semibold">
+                Search for repositories
+              </span>
+              <span className="text-left">Search for specific repositories to track on your workspace.</span>
+            </button>
+            <button
+              className="flex flex-col text-light-slate-12 p-8 border rounded-lg focus-visible:!border-green-800 focus-visible:!ring-green-100"
+              onClick={onImportOrg}
+            >
+              <BiBarChartAlt2 className="text-purple-800 mb-2" />
```
Comment (nickytonline) [inline components/Workspaces/TrackedRepoWizardStep1.tsx:45]: @isabensusan, same as the tracked repos table PR, I can't find the exact bar chart icon you have in Figma.

File: components/Workspaces/TrackedRepoWizardStep1.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+export const TrackedRepoWizard = ({
+  onAddToTrackingList,
+  onSearchRepos,
+  onImportOrg,
+  onCancel,
+  trackedReposCount,
+}: TrackedRepoWizardProps) => {
+  return (
+    <Card className="p-0 w-max max-w-3xl">
```
Comment (nickytonline) [inline components/Workspaces/TrackedRepoWizardStep1.tsx:23]: The text was not passing colour contrast checks as it's below light-slate-12, so I'm currently using the default text colour defined on the body.

**Review by jpmcb — APPROVED** [review]
🧙🏼‍♂️ 🧙🏼‍♂️ 🧙🏼‍♂️ 🧙🏼‍♂️ 🧙🏼‍♂️

**Review by brandonroberts — APPROVED** [review]
🍕

**Pull request merged**